src/pykeen/models/base.py

```diff
@@ -490,6 +556,89 @@ def predict_scores_all_heads(
             scores = torch.sigmoid(scores)
         return scores
 
+    def score_all_triples(
+        self,
+        k: Optional[int] = None,
+        batch_size: int = 1,
+    ) -> torch.LongTensor:
+        """
+        Compute scores for all triples, optionally returning only the k highest scoring.
+
+        .. note ::
+            This operation is very expensive for reasonably-sized knowledge graphs.
```

Comment:
@cthoyt Please check whether this is the correct way to get this nicely displayed in the doc :wink:

Reply:
There's a bit of a typo - you need to make sure the `::` comes right after the directive like in:

```
# wrong:
.. note ::

# right:
.. note::
```

If you can manage to fit it all on one line, it can come directly after the `::`, otherwise, there should be a blank like and an ident
```
.. note::

    block text
    goes here
```

Reply:
I fixed this in 83d6309 and also updated the autogeneration of docs in 8bb8742 so this actually gets turned into a page in the reference